Automattic/newspack-popups
# feat: new option to not display an overlay background

### All Submissions:






### Changes proposed in this Pull Request:



A follow-up to #632, which allows editors to make the overlay background invisible. This is a small feature to let editors optionally render no overlay background at all. This is mainly relevant for "toaster" like overlays that appear at the edge of the viewport, leaving most of the page content visible. Currently, if an overlay prompt is visible, the reader must dismiss it in order to interact with the page underneath. With this change, the reader is able to interact with the page without dismissing the prompt first.



### How to test the changes in this Pull Request:

1. Publish some overlay prompts with various overlay options.
2. Check out this branch and confirm that the overlay prompts continue to behave exactly as before.
3. Edit an overlay prompt and confirm there's a new toggle option under the Colors sidebar, which is ON by default:

<img width="279" alt="Screen Shot 2022-05-13 at 4 54 27 PM" src="https://user-images.githubusercontent.com/2230142/168399364-dbf8f48e-671b-432b-9ab4-1c1b105b08b3.png">

4. Turn it OFF and update.
5. View the prompt on the front-end and confirm that the prompt is shown with no background, similar in appearance to a 0% overlay, except you should also be able to interact with page elements underneath the overlay without first dismissing it.
6. Confirm that you're still able to interact with elements _inside_ the prompt, as well.

### Other information:
